lib/Database.js

```diff
@@ -148,6 +151,45 @@ var Database = module.exports = Class.extend({
     });
   },
 
+  breakStringIntoArrays: function (orders) {
+    var arr = orders.split(',');
+    var set = [];
+    for (var i in arr) {
+      set.push(arr[i].trim().split(' '));
+    }
+    return set;
+  },
+
+  /**
+   * Builds the order by query segment
+   */
+  getOrderBy: function (model, orders) {
+    var db = this;
+    if (!orders || (orders && orders.length === 0)) return '';
+
+    if (typeof orders == 'string') {
+      return db.getOrderBy(model, db.breakStringIntoArrays(orders));
+    }
+
+    var query = '';
+    for (var i in orders) {
```

Comment:
for..in iteration is slower, and it's also susceptible to Object.prototype changes.